### PR TITLE
Installation/index: add section for directory info

### PR DESCRIPTION
**Pull request by Rosi2143**
often in these pages directories are required and there
is no info how to get them yet.

e.g. where does $OPENHAB_CONF point to.

Signed-off-by: Michael Roßner <Schrott.Micha@web.de>

### CODE REVIEW

**Comment by Confectrician** [comment]
Hey @Rosi2143 thanks for starting this.
We have an issue for this #189 and a basic table in https://github.com/openhab/openhab-docs/issues/189#issuecomment-360506100

We should check if we cann/should combine this table with your edits.
For sure the table will be outdated for the openhab 2 paths.

**Comment by Confectrician** [comment]
Having a preview available i would say this suits most of the cases already, at least when openhab.cli is available.
I would doubt that on windows installations but windows has its own file locations table in place.

So we maybe should at least add some clarifiaction on where to use the openhab-cli command (think of a beginner reading the docs first time)

File: installation/index.md [hunk 1]
```diff
@@ -87,6 +87,36 @@ Before you can start, two decisions have to be made:
 
 Please follow the instructions in the installation article matching your platform (see the menu to the left).
 
+### getting installation info
```
Comment (Confectrician) [inline installation/index.md:90]: Should be `### Getting installation info`

File: installation/index.md [hunk 2]
```diff
@@ -87,6 +87,36 @@ Before you can start, two decisions have to be made:
...
+
+For many actions you need to know the pathes where openHAB checks for files, or where you need to install additional packages. To find out where these locations are, e.g. where `$OPENHAB_CONF` points to, use the [console](https://www.openhab.org/docs/administration/console.html):
```
Comment (Confectrician) [inline installation/index.md:92]: Please use one line per sentence.

File: installation/index.md (hunk 1 above)
Comment (Rosi2143) [reply installation/index.md]: Done.

File: installation/index.md (hunk 2 above)
Comment (Rosi2143) [reply installation/index.md]: Done

**Comment by Rosi2143** [comment]
Changes done

**Review by Confectrician — APPROVED** [review]
Thanks. 👍

**Comment by Confectrician** [comment]
I would say we can close #189 too with this PR in place.

**Pull request merged**